test/js/third_party/postgres/postgres.test.ts

```diff
@@ -19,6 +19,33 @@ describe("pg", () => {
       pool.end();
     }
   });
+
+  it("should execute big query and end connection", async () => {
+    const client = new Client({
+      connectionString: CONNECTION_STRING,
+      ssl: { rejectUnauthorized: false },
+    });
+
+    async function execute() {
+      try {
+        await client.connect();
+        const res = await client.query(`SELECT * FROM users LIMIT 1000`);
+        expect(res.rows.length).toBeGreaterThanOrEqual(300);
+        await client.end();
+        return "success";
+      } catch (e) {
+        return e;
+      }
+    }
+
+    async function timeout() {
+      await Bun.sleep(5000);
```

Comment:
This will always execute after the test has completed 

Which is not intended I think

Reply:
Yeah, will just rely on the test already implemented timeout:
https://github.com/oven-sh/bun/pull/6487/commits/a96898d20397ead25587bcdace5d09b02ab04077